cosign/download/action.yaml

```diff
@@ -0,0 +1,18 @@
+name: "Piwik PRO cosign download"
```

Comment:
Have you considered using of the official [cosign action](https://github.com/sigstore/cosign-installer)?

Reply:
Yes, Jacek refused any external action in container signing process